### ISSUE TITLE
An error occured while trying to run opener (1) - SumatraPDF 3.1.2

### ISSUE THREAD

**Issue by moospit** [thomasjo/atom-latex#282]
I use `latex` with SumatraPDF as viewer. When compiling a document the first time SumatraPDF opens up just fine. Rebuilding (no matter if changes made to the document) results in the message:

    An error occured while trying to run opener (1)

If changes have been made SumatraPDF displays them just fine, but i don't get any other errors or warnings from the build process. SyncTex seems to work just fine.

Versions:
 * `latex`: 0.38.1
 * `atom`: 1.11.2
 * `SumatraPDF`: 3.1.2

**Comment by thomasjo** [comment]
I think this is a known problem. We're aware that the SumatraPDF opener is a bit wonky, but that is, as far I as know, caused by weirdness with SumatraPDF itself. I guess until we figure out how to work around the issues with SumatraPDF, we might have to silently ignore the returned exit codes (or possibly loging them as info messages instead of just silently ignoring them).

/cc @yitzchak

**Comment by yitzchak** [comment]
@thomasjo I have cleaned up `SumatraOpener` and tried every combination of arguments I can think of. Nothing makes a difference. Currently, the error is ignored in master. It is possible to turn it into an info message, but not easily because the error is generated after `latex.log.groupEnd` is called. Therefore, recording the error (whatever the severity) clobbers the current messages resulting from `LogParser`.

**TL;DR** It's fixed in the next version by ignoring Sumatra's weirdness.

**Comment by karyon** [comment]
for the record, as mentioned in https://github.com/thomasjo/atom-latex/issues/250, i get this whenever the document compiles *without* errors and clicking on the "latex log". when there *are* errors, clicking on it actually opens a small pane that shows errors and warnings.

**Comment by yitzchak** [comment]
This we be resolved in the next version.
